src/ui/public/courier/saved_object/ui/saved_object_save_as_checkbox.html

```diff
@@ -0,0 +1,9 @@
+<div ng-hide="!savedObject.id">
+  <div ng-hide="!savedObject.isTitleChanged() || savedObject.copyOnSave" class="localDropdownWarning">
+    Note, saving a {{savedObject.getDisplayName()}} with a new name will no longer automatically create a copy.
```

Comment:
If I were to read this as someone who's new to Kibana (and lacks the context of how the Save functionality currently works), I don't think I'd be able to figure out what "no longer" means. I think Spencer's message, though way more verbose, was a little clearer because it explicitly explained how the previous version of Kibana worked.